vpr/src/route/connection_box_lookahead_map.cpp

```diff
@@ -537,6 +575,83 @@ float ConnectionBoxMapLookahead::get_expected_cost(
     }
 }
 
+static int manhattan_distance(const t_rr_node& node, int x, int y) {
+    int node_center_x = (node.xhigh() + node.xlow()) / 2;
+    int node_center_y = (node.yhigh() + node.ylow()) / 2;
+    return abs(node_center_x - x) + abs(node_center_y - y);
+}
+
+static BoundingBox<int> bounding_box_for_node(const t_rr_node& node) {
+    return BoundingBox<int>(node.xlow(), node.ylow(),
+                            node.xhigh(), node.yhigh());
+}
+
+void ConnectionBoxMapLookahead::find_start_inodes_for_segment_types(int num_segments) {
```

Comment:
Why is the function a method of `ConnectionBoxLookahead` at all?  `bounding_box_for_segment_` and `start_inode_for_segment_` both seem like temporary variables only used during `compute`?

Reply:
It wasn't clear at first that `compute` is only called once, but yeah, I should get rid of these.